utilities/ipplan-import.py

```diff
@@ -0,0 +1,181 @@
+"""
+NIPAP import script for IPplan data.
+
+This script uses the standard export files base.txt and ipaddr.txt from Ipplan.
+"""
+import configparser
+import os
+# use local pynipap, useful if we are developing
+import sys
+sys.path.append('../pynipap')
+import pynipap
+import csv
+import ipaddress
+
+
+def add_ip_to_net(networks, host):
+    """
```

Comment:
Join this line with the next to follow PEP8 / PEP257 style guide on docstrings (https://www.python.org/dev/peps/pep-0008/#documentation-strings / https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).. and do so for the rest of the file :)